src/browsers/index.ts

```diff
@@ -328,6 +325,23 @@ export class BrowserManager {
     }
   }
 
+  public async killSessions(target: string): Promise<void> {
```

Comment:
We should probably throw a 404 or something if the session isn't found. This way folks will know if it was successful or not